api/v1/Game.php

```diff
@@ -805,15 +803,11 @@ public function GetGameDetails(): array|PromiseInterface
             }
 
             $realtimePerEra = explode(",", $state["game_planning_era_realtime"]);
-            // todo: division by zero.
-            $currentEra = intval(floor($state["game_currentmonth"] / $state["game_planning_gametime"]));
-            $realtimePerEra[$currentEra] = $state["game_planning_realtime"];
-            $secondsPerMonthCurrentEra = round($state["game_planning_realtime"] / $state["game_eratime"]);
-            $monthsRemainingCurrentEra = $state["game_eratime"] - $state["game_planning_monthsdone"];
-            $totalRemainingTime = $monthsRemainingCurrentEra * $secondsPerMonthCurrentEra;
+            $totalRemainingTime = intval($state["game_planning_realtime"]);
```

Comment:
The lines above had two problems: 1. They actually used the wrong table column.  2. They were overcomplicating the calculation. 

The only thing that needs to be done here is calculate how long the entire game session has been planned to take by the admin. For that you just need to add the remaining time left in the current era with the time left in future eras.

Reply:
what do you mean by "to take by the admin" ?
It is quite funny that the "totalRemaingTIme" is now just a assignment from the database field value "game_planning_realtime". 
Whereas before it was a bunch of calculation . Does not compare really... only if I understand each field value indepth , I can see if these changes make sense. At least I can see if a lot less complicated, it is just hard to for me tell right now if it is still correct. 
But since the old code was not fully understandable either (I would really need some value examples to see what going on , or step through it)

So for now I tend to just approve the changes, and just keep on eye on any effect . These values would be seen the upperright corner of the game ? the time line right ?

Reply:
has been explained